dotnet/name-service/Controllers/NameController.cs

```diff
@@ -44,28 +47,29 @@ private static string GetYearEndpoint()
         [HttpGet]
         public async Task<string> GetAsync()
         {
-            var current = Activity.Current;
-            current?.AddTag("apple", 1);
-            current?.AddBaggage("avocado", "12");
+            var currentSpan = Tracer.CurrentSpan;

```

Comment:
Why not use `_tracer` here?

Reply:
explained in slack, let's 🚢 🇮🇹

Reply:
`_tracer` does not provide current span 😞